platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -313,6 +313,8 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)mapViewDidStopLocatingUser:(MGLMapView *)mapView;
 
+- (MGLUserLocationStyle)mapViewStyleForUserLocationView:(MGLMapView *)mapView;
```

Comment:
This method should be documented. If we don't want the documentation to appear in the API guides, prepend the documentation entry with `:nodoc:`.

Reply:
This was a work in progress. I added the documentation.